docs/conceptual/cmake-packages.rst

```diff
@@ -140,6 +144,11 @@ any C or C++ compiler can be used. The ``find_package(hip)`` provides the
     add_executable(MyApp ...)
     target_link_libraries(MyApp PRIVATE hip::host)
 
+When mixing ``CXX`` sources using the host API only and HIP sources with
```

Comment:
@LisaDelaney this sentence needs a second look

Reply:
- Do we need to say what "CXX" is in the first instance?
- As for rephrasing  (it's a confusing/dense sentence), how do you feel about this? "When mixing ``CXX`` sources that use 1) host API only and 2) HIP with device code, link only to `hip::host`. If the HIP sources don't have the `.hip` extension, you must use `set_source_files_properties(<hip_sources>... PROPERTIES LANGUAGE HIP)`."

Reply:
Rephrased.